src/ts/player-events.d.ts

```diff
@@ -12,87 +12,79 @@ declare namespace bitmovin {
-      ON_TIME_SHIFT: EVENT;
-      ON_TIME_SHIFTED: EVENT;
-      ON_UNMUTED: EVENT;
-      ON_VIDEO_ADAPTATION: EVENT;
-      ON_VIDEO_QUALITY_CHANGED: EVENT;
-      ON_VIDEO_DOWNLOAD_QUALITY_CHANGE: EVENT;
-      ON_VIDEO_DOWNLOAD_QUALITY_CHANGED: EVENT;
-      ON_VIDEO_PLAYBACK_QUALITY_CHANGED: EVENT;
-      ON_VOLUME_CHANGED: EVENT;
-      ON_VR_ERROR: EVENT;
-      ON_VR_MODE_CHANGED: EVENT;
-      ON_VR_STEREO_CHANGED: EVENT;
-      ON_WARNING: EVENT;
-      ON_PICTURE_IN_PICTURE_ENTER: EVENT;
-      ON_PICTURE_IN_PICTURE_EXIT: EVENT;
-      ON_AIRPLAY_AVAILABLE: EVENT;
-      ON_VR_VIEWING_DIRECTION_CHANGE: EVENT;
-      ON_VR_VIEWING_DIRECTION_CHANGED: EVENT;
-      ON_DESTROY: EVENT;
-      ON_AD_BREAK_STARTED: EVENT;
-      ON_AD_BREAK_FINISHED: EVENT;
-      ON_PLAYBACK_SPEED_CHANGED: EVENT;
+      AdBreakStarted: EVENT;
+      AdBreakFinished: EVENT;
+      AdClicked: EVENT;
+      AdError: EVENT;
+      AdFinished: EVENT;
+      AdLinearityChanged: EVENT;
+      AdManifestLoaded: EVENT;
+      AdScheduled: EVENT;
+      AdSkipped: EVENT;
+      AdStarted: EVENT;
+      AudioAdaptation: EVENT;
+      AudioChanged: EVENT;
+      AudioAdded: EVENT;
+      AudioRemoved: EVENT;
+      AudioQualityChanged: EVENT;
+      AudioDownloadQualityChange: EVENT;
+      AudioDownloadQualityChanged: EVENT;
+      AudioPlaybackQualityChanged: EVENT;
+      CastAvailable: EVENT;
+      CastStart: EVENT;
+      CastStarted: EVENT;
+      CastWaitingForDevice: EVENT;
+      CastStopped: EVENT;
+      CueParsed: EVENT;
+      CueEnter: EVENT;
+      CueUpdate: EVENT;
+      CueExit: EVENT;
+      Destroy: EVENT;
+      DownloadFinished: EVENT;
+      DVRWindowExceeded: EVENT;
+      Error: EVENT;
+      FullscreenEnter: EVENT;
+      FullscreenExit: EVENT;
+      Metadata: EVENT;
+      MetadataParsed: EVENT;
+      Muted: EVENT;
+      Paused: EVENT;
+      PeriodSwitch: EVENT;
+      PeriodSwitched: EVENT;
+      Play: EVENT;
+      Playing: EVENT;
+      PlaybackFinished: EVENT;
+      PlaybackSpeedChanged: EVENT;
+      PlayerResize: EVENT;
+      Ready: EVENT;
+      Seek: EVENT;
+      Seeked: EVENT;
+      SegmentPlayback: EVENT;
+      SegmentRequestFinished: EVENT;
+      SourceLoaded: EVENT;
+      SourceUnloaded: EVENT;
+      StallStarted: EVENT;
+      StallEnded: EVENT;
+      SubtitleAdded: EVENT;
+      SubtitleChanged: EVENT;
+      SubtitleRemoved: EVENT;
+      TimeChanged: EVENT;
+      TimeShift: EVENT;
+      TimeShifted: EVENT;
+      Unmuted: EVENT;
+      VideoAdaptation: EVENT;
+      VideoQualityChanged: EVENT;
+      VideoDownloadQualityChange: EVENT;
+      VideoDownloadQualityChanged: EVENT;
+      VideoPlaybackQualityChanged: EVENT;
+      VolumeChanged: EVENT;
+      VRStereoChanged: EVENT;
+      Warning: EVENT;
+      PictureInPictureEnter: EVENT;
+      PictureInPictureExit: EVENT;
+      AirplayAvailable: EVENT;
+      VRViewingDirectionChange: EVENT;
+      VRViewingDirectionChanged: EVENT;
```

Comment:
There's obviously a few events missing here. Have they been removed on purpose?

Reply:
Some Events were removed with V8:
`onShowControls`, `onHideControls` were completely removed
`onCastPlaying`, `onCastPlaybackFinished`, `onCastPaused`, `onCastTimeUpdated` have been replaced with the "normal" player events a while ago and have now been removed